test/unit/errors/error-collector.test.js

```diff
@@ -2370,3 +2370,76 @@ test('When using the async listener', (t) => {
     })
   }
 })
+
+tap.test('_processErrors', (t) => {
+  let transaction = null
+  let errorCollector = null
+
+  t.beforeEach(() => {
+    t.context.agent = helper.loadMockedAgent({
+      attributes: {
+        enabled: true
+      }
+    })
+
+    transaction = new Transaction(t.context.agent)
+    transaction.url = '/'
+    errorCollector = t.context.agent.errors
+  })
+
+  t.afterEach(() => {
+    helper.unloadAgent(t.context.agent)
+  })
+
+  t.test('invalid errorType should return no iterableProperty', (t) => {
+    const errorType = 'invalid'
+    const result = errorCollector._getIterableProperty(transaction, errorType)
+
+    t.equal(result, null)
+    t.end()
+  })
```

Comment:
Given the previous suggestion, this test would look like:

```suggestion
  t.test('invalid errorType should return no iterableProperty', (t) => {
    const { errorCollector, transaction } = t.context
    const errorType = 'invalid'
    const result = errorCollector._getIterableProperty(transaction, errorType)

    t.equal(result, null)
    t.end()
  })
```